### PR TITLE
pin s3-synapse-sync v1.4.0

### PR DESCRIPTION
**Pull request by clarisse-lau**
Pin lambda to use v1.4.0

### CODE REVIEW

File: config/prod/s3-synapse-sync.yaml [hunk 1]
```diff
@@ -21,3 +21,6 @@ parameters:
     }
   KmsDecryptPolicyArn: !stack_output_external "htan-synapse-sync-kms-key::KmsDecryptPolicyArn"
   BucketNamePrefix: "htan-dcc-*"
+  DockerImage: "clarisselau/minerva-batch"
+  Subnets: "subnet-1a2b3c4d,subnet-1a2b3c4d"
```
Comment (zaro0508) [inline config/prod/s3-synapse-sync.yaml:25]: You can use [sceptre stack_output_external reference](https://sceptre.cloudreach.com/2.2.1/docs/resolvers.html#stack-output-external) to access our VPC info, like so..

```
Subnets:
  - !stack_output_external computevpc::PrivateSubnet
  - !stack_output_external computevpc::PrivateSubnet1
  - !stack_output_external computevpc::PrivateSubnet2
```

which is NOT a string of subnets but rather a list of subnets (i.e. ["subnet-11111","subnet-22222","subnet-3333"])

File: config/prod/s3-synapse-sync.yaml [hunk 2]
```diff
@@ -21,3 +21,6 @@ parameters:
...
+  VpcID: "vpc-1a2b3c4d"
```
Comment (zaro0508) [inline config/prod/s3-synapse-sync.yaml:26]:
```
VpcID: !stack_output_external computevpc::VPCId
```

**Review by zaro0508 — CHANGES_REQUESTED** [review]
Would you mind separating these into 2 PRs, one for updating s3-synapse-sync.yaml and one for updating all the buckets?

File: config/prod/s3-synapse-sync.yaml (hunk 1 above)
Comment (clarisse-lau) [reply config/prod/s3-synapse-sync.yaml]: thanks! added the subnets

File: config/prod/s3-synapse-sync.yaml (hunk 2 above)
Comment (clarisse-lau) [reply config/prod/s3-synapse-sync.yaml]: updated

**Review by zaro0508 — APPROVED** [review]

**Review by zaro0508 — CHANGES_REQUESTED** [review]
PR looks good, however there are linter errors. please fix.

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**